src/picongpu/include/simulation_defines/param/fieldBackground.param

```diff
@@ -33,7 +37,23 @@ namespace picongpu
 
         /* We use this to calculate your SI input back to our unit system */
         const float3_64 unitField;
-        HDINLINE fieldBackgroundE( const float3_64 unitField ) : unitField(unitField)
+
+        /* TWTS E-fields need to be initialized on host, so they can look up global grid dimensions. */
```

Comment:
that is a nice example.

the only thing: it should not be part of the `defaul` files in `src/picongpu/include/simulation_defines/`.
please just move it to either `examples/Bunch` or `examples/LaserWakefield` and we can add a specific test there that activates and uses it.